### DISCUSSION TITLE
Add option for frequently updated maps

### DISCUSSION THREAD

**Discussion by jeffboy70** [organicmaps/organicmaps#1181] [Ideas]
Add an option for the OSM map to update one every 12 hours or so, or just have button that updates the map to the most recent version.  Lots of people (including myself) live in areas where the map is constantly being improved and having even a week old map could be missing out on a lot.  I think OsmAnd has a feature like this called OsmAnd Live?

**Comment by biodranik** [comment]
Your idea is great! But our current implementation is not perfect.

We generate all maps at once. It takes now around 3 days minimum if everything goes well, plus tests/checks and then deployment to servers.

Our nearest goal is to update maps at least weekly. Then, if we have enough time and money resources, we can start complex refactorings to make updates even faster.
